### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yoelhor**
Adding b2clogin.com and advance topics

### CODE REVIEW

**Comment by jmprieur** [comment]
@negoe @DarylThayil @sameerag 
can you please review and merge if you are happy with this PR?

File: README.md [hunk 1]
```diff
@@ -166,8 +166,29 @@ Your resulting code should look as follows:
 5. Click on the **Call Web API** and see the textual representation of the JSON object that is returned. Make sure your Node.js Web API sample is still running on port 5000. 
 6. Sign out by clicking the **Logout** button.  
 
+## Advance topics
```
Comment (jmprieur) [inline README.md]: Advanced?

Comment (yoelhor) [reply README.md]: I can change the title to **optional**

Comment (DarylThayil) [inline README.md]: @sameerag  / @pkanher617  review for accuracy

Comment (sameerag) [reply README.md]: @negoe @jmprieur Should we also add sid/account params which we support as am alternative to login_hint here or is it too much information?

Comment (jmprieur) [reply README.md]: @sameerag : if sid works better, then yes

Comment (sameerag) [reply README.md]: It's mostly the new API: We have sid || login_hint || account (object) and we recommend only one option passed. We generate the hints based on this info and there is a priority on which hints we pick and discard depending on the data available.

I will work with @negoe to document this clearly.

**Comment by jennyf19** [comment]
@yoelhor @sameerag @DarylThayil @jmprieur Can this be merged?

File: README.md [hunk 2]
```diff
@@ -166,8 +166,29 @@ Your resulting code should look as follows:
 5. Click on the **Call Web API** and see the textual representation of the JSON object that is returned. Make sure your Node.js Web API sample is still running on port 5000. 
 6. Sign out by clicking the **Logout** button.  
 
+## Optional
+- [Configure application to use b2clogin.com](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-b2c-overview#configure-application-to-use-b2clogincom)
```
Comment (jmprieur) [inline README.md:170]: you might want to remove "/en-us" (so that customers get a localized page on docs.ms)

Comment (jmprieur) [inline README.md:170]: Is it normal that the documentation proposes an authority whic his not a b2clogin.com ? 
https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-b2c-overview#step-3-configure-authentication but `https://login.microsoftonline.com/tfp/<your-tenant-name>.onmicrosoft.com/<your-sign-in-sign-up-policy>` ?

**Review by jmprieur — APPROVED** [review]
LGTM. Thanks @jennyf19

File: README.md (hunk 1 above)
Comment (sameerag) [reply README.md]: @negoe Is working on the adjacent doc for hints. We can merge this and append the link to that document later if this is delayed too much.

**Review by sameerag — APPROVED** [review]
Approved. Hints doc will follow from @negoe

File: README.md (hunk 2 above)
Comment (mmacy) [reply README.md:170]: Nope, we should always promote b2clogin.com as the happy path.

**Comment by jennyf19** [comment]
thanks for the contribution @yoelhor

File: README.md (hunk 2 above)
Comment (negoe) [reply README.md:170]: @jmprieur I will update the doc.

**Pull request merged**